qplist.pro

```diff
@@ -0,0 +1,14 @@
+QT += xml
```

Comment:
Ok, viewing from that angle, I'd say it is more beneficial for a project like this to be called qtplist. I will provide you with a changeset.